scripts/src/javatests/org/oppia/android/scripts/maven/GenerateMavenDependenciesListTest.kt

```diff
@@ -440,33 +514,119 @@ class GenerateMavenDependenciesListTest {
         "${tempFolder.root}/scripts/assets/maven_dependencies.pb"
       )
     )
-    val textprotoContent = textprotoFile.readAsJoinedString()
-    assertThat(textprotoContent).contains(
-      """
-      maven_dependency {
-        artifact_name: "androidx.databinding:databinding-adapters:3.4.2"
-        artifact_version: "3.4.2"
-        license {
-          license_name: "The Apache License, Version 2.0"
-          original_link: "https://www.apache.org/licenses/LICENSE-2.0.txt"
-          scrapable_link {
-            url: "https://www.apache.org/licenses/LICENSE-2.0.txt"
-          }
-        }
-      }
-      maven_dependency {
-        artifact_name: "com.google.protobuf:protobuf-lite:3.0.0"
-        artifact_version: "3.0.0"
-        license {
-          license_name: "Simplified BSD License"
-          extracted_copy_link {
-            url: "https://local-copy/bsd-license"
-          }
-        }
-      }
-      """.trimIndent()
-    )
+
     assertThat(outContent.toString()).contains(SCRIPT_PASSED_MESSAGE)
+
+    val outputMavenDependencyList = getProto(
+      textprotoFile,
+      MavenDependencyList.getDefaultInstance()
+    )
+
+    val dependency1 = outputMavenDependencyList.mavenDependencyList[0]
+    assertDependency(
+      dependency = dependency1,
+      artifactName = DEP_WITH_DIRECT_LINK_ONLY_LICENSE,
+      artifactVersion = FIREBASE_ANALYTICS_VERSION,
+    )
+    val licenseForDependency1 = dependency1.licenseList[0]
+    verifyLicenseHasDirectLinkOnlyVerifiedLink(
+      license = licenseForDependency1,
+      originalLink = "https://developer.android.com/studio/terms.html",
+      licenseName = "Android Software Development Kit License",
+      verifiedLink = "https://developer.android.com/studio/terms.html"
+    )
+    val dependency2 = outputMavenDependencyList.mavenDependencyList[1]
+    assertDependency(
+      dependency = dependency2,
+      artifactName = DEP_WITH_NO_LICENSE,
+      artifactVersion = PROTO_LITE_VERSION,
+    )
+    val license1ForDependency2 = dependency2.licenseList[0]
+    verifyLicenseHasExtractedCopyVerifiedLink(
+      license = license1ForDependency2,
+      originalLink = "",
+      licenseName = "Simplified BSD License",
+      verifiedLink = "https://local-copy/bsd-license"
+    )
+  }
+
+  private fun verifyLicenseHasScrapableVerifiedLink(
+    license: License,
+    originalLink: String,
+    licenseName: String,
+    verifiedLink: String,
+  ) {
+    val scrapableLink = ScrapableLink.newBuilder().setUrl(verifiedLink).build()
+    assertThat(license.licenseName).isEqualTo(licenseName)
+    assertThat(license.scrapableLink).isEqualTo(scrapableLink)
+    assertThat(license.originalLink).isEqualTo(originalLink)
+    assertThat(license.isOriginalLinkInvalid).isEqualTo(false)
```

Comment:
Prefer:

```suggestion
    assertThat(license.isOriginalLinkInvalid).isFalse()
```

Ditto elsewhere, and for the true case.

Reply:
Done.